package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "import-sort-style-module-alias-grouping",
-  "version": "0.1.6",
+  "version": "0.1.7",
```

Comment:
We'll have to cut a new release rather than revert this change and update our app to use `0.1.8`. So, I would remove this file from the changeset